.github/workflows/phpunit.yml

```diff
@@ -0,0 +1,99 @@
+name: PHPUnit tests
+
+on: pull_request
+
+env:
+  WP_TRAVISCI: "phpunit"
+  COMPOSER_ROOT_VERSION: "dev-master"
+
+jobs:
+  phpunit:
+    name: "PHP: ${{ matrix.php }} WP: ${{ matrix.wp }}"
+    runs-on: ubuntu-latest
+    strategy:
+      fail-fast: false
+      matrix:
+        php: ["5.6", "7.0", "7.2", "7.3", "7.4snapshot"]
+        wp: ["master", "latest", "previous"]
+        exclude:
+        # excludes node 4 on macOS
+        - php: "7.0"
+        - php: "7.2"
+        - wp: "latest"
+
+    steps:
+      - uses: actions/checkout@v2
+
+      - name: Setup PHP
+        uses: shivammathur/setup-php@v2
+        with:
+          php-version: ${{ matrix.php }}
+          tools: composer, phpunit:v7
```

Comment:
Does including "phpunit:v7" here install a phar, or install standalone somehow (e.g. via composer)?

If the former, it's going to break for the PHP 8 run. Better probably to just not install it here in the first place in that case, do a step to `composer global require` it instead (like in setup-travis.sh).

Reply:
👍